books/free-programming-books-langs.md

```diff
@@ -1457,6 +1458,11 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Modelica by Example](http://book.xogeny.com)
 
 
+### MongoDB
+
+* [The Little MongoDB book](https://www.openmymind.net/mongodb.pdf)  (PDF)
```

Comment:
Add authoring attributions to satisfy CC license is a must

Reply:
sure, will make necessary changes

Reply:
Already listed. Check and update resource metadata

https://github.com/EbookFoundation/free-programming-books/blob/83beecd381eabcdf8e54782a07358792e31b3644/books/free-programming-books-langs.md#L1520